<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite parallel="classes" name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" parallel="classes" name="Test(failed)">
    <classes>
      <class name="remoteTesting.dockerValidation.chromeTest1">
        <methods>
          <include name="test1"/>
        </methods>
      </class> <!-- remoteTesting.dockerValidation.chromeTest1 -->
      <class name="remoteTesting.dockerValidation.chromeTest2">
        <methods>
          <include name="test2"/>
        </methods>
      </class> <!-- remoteTesting.dockerValidation.chromeTest2 -->
      <class name="remoteTesting.dockerValidation.chromeTest3">
        <methods>
          <include name="test3"/>
        </methods>
      </class> <!-- remoteTesting.dockerValidation.chromeTest3 -->
      <class name="remoteTesting.dockerValidation.stopDocker">
        <methods>
          <include name="stopAndDelete"/>
        </methods>
      </class> <!-- remoteTesting.dockerValidation.stopDocker -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
